docs/fixing-warnings.md

```diff
@@ -0,0 +1,189 @@
+
+# Trim warnings in .NET 6
+
+In .NET Core 3.1 and 5.0 we introduced trimming as a new preview feature for self-contained .NET
+core applications. Conceptually the feature is very simple: when publishing the application the
+.NET SDK analyzes the entire application, including the .NET framework, and removes all unused
```

Comment:
I'm not sure what "including the .NET framework" is intended to explain

Reply:
I meant to emphasize that we're trimming away the core framework, but it's probably not necessary.